variation/tokenizers/deletion_base.py

```diff
@@ -77,6 +77,10 @@ def _get_parts(self, parts):
         else:
             end_pos_del = None
 
+        if start_pos_del and end_pos_del:
```

Comment:
this method is missing type annotations fyi

Reply:
I always think I did a good job at making sure I add type hints / docs and then I look at your comments and am very wrong. One day I will be able to open a PR without missing types